### PR TITLE
change .ssh/config to .ssh/known_hosts

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: intro-HPC/ch_troubleshooting.tex [hunk 1]
```diff
@@ -245,7 +245,7 @@ \section{Security warning about invalid host key}
 \end{prompt}
 
 You will need to remove the line it's complaining about (in the example, line 21).
-To do that, open \lstinline|~/.ssh/config| in an editor, and remove the line. This
+To do that, open \lstinline|~/.ssh/known_hosts| in an editor, and remove the line. This
 results in \lstinline|ssh| ``forgetting'' the system you are connecting to.
```
Comment (boegel) [inline intro-HPC/ch_troubleshooting.tex:249]: Actually, a better suggestion is to run `ssh-keygen -R name.of.host`, maybe mention that too as an alternative?

Comment (hajgato) [reply intro-HPC/ch_troubleshooting.tex:249]: does that work with mac?

Comment (boegel) [reply intro-HPC/ch_troubleshooting.tex:249]: yes

Comment (hajgato) [reply intro-HPC/ch_troubleshooting.tex:249]: fixed

File: intro-HPC/ch_troubleshooting.tex [hunk 2]
```diff
@@ -245,9 +245,16 @@ \section{Security warning about invalid host key}
...
 
+Alternatively you can use the command shown by the warning under
+\lstinline|remove with:| that should be something like this:
+
+\begin{prompt}
+sh-keygen -f "~/.ssh/known_hosts" -R "%\loginnode{}%"
```
Comment (boegel) [inline intro-HPC/ch_troubleshooting.tex:255]: typo: should be `ssh-keygen`

**Review by boegel — CHANGES_REQUESTED** [review]

File: intro-HPC/ch_troubleshooting.tex (hunk 2 above)
Comment (hajgato) [reply intro-HPC/ch_troubleshooting.tex]: the devil is in `CTRL-C` `CTRL-V`

**Review by boegel — APPROVED** [review]

**Pull request merged**